internal/exchange/websocket/manager.go

```diff
@@ -157,10 +157,10 @@ type ManagerSetup struct {
 	RateLimitDefinitions request.RateLimitDefinitions
 }
 
-// ConnectionWrapper contains the connection setup details to be used when
+// connectionWrapper contains the connection setup details to be used when
 // attempting a new connection. It also contains the subscriptions that are
 // associated with the specific connection.
-type ConnectionWrapper struct {
+type connectionWrapper struct {
 	// Setup contains the connection setup details
 	Setup *ConnectionSetup
```

Comment:
I don't think these fields need exporting now that it is an internal struct with it never being a return type. I will forgive this being resolved to avoid additional changes in this pr.... But you started it when you privatised it

Reply:
Agreed. Prefer strong boundaries and defensive coding.
Removed comments that just stutter through the same var and types.
Removed the TODO; FeatureDrivenDevelopment over ImplementationHints. If this needs to be a slice cthulhu will ensure it happens.

Fixed https://github.com/gbjk/gocryptotrader/commit/6fb51f77e0